src/components/GameView/SpectatorListMenu.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-menu v-model="spectatingMenu">
+    <!-- Activator -->
+    <template #activator="{ props }">
+      <v-btn
+        data-cy="spectate-list-button"
+        class="pl-2"
+        id="spectating-menu-activator"
+        v-bind="props"
+        icon
+        variant="text"
+      >
+        <v-icon class="mr-1" icon="mdi-eye-outline" size="large" />
+        <span class="pr-2" v-if="!$vuetify.display.xs">{{ spectatingUsers.length }}</span>
+      </v-btn>
+    </template>
+    <!-- Menu -->
+    <v-list class="pl-2 pr-2" id="spectatorList" data-cy="spectate-list-menu">
+      <v-list-title>Spectators</v-list-title>
```

Comment:
This component has been renamed to v-list-item-title
```suggestion
      <v-list-item-title>Spectators</v-list-item-title>
```